geesRecorderClient/Client/Pages/Attendance/Mark.razor

```diff
@@ -0,0 +1,93 @@
+@page "/attendance/mark/{ProjectId:int}"
```

Comment:
Not bad at all. You're adapting pretty well. I'll do the rest of the implementation for this page